code/controllers/subsystems/vote.dm

```diff
@@ -117,7 +117,9 @@ SUBSYSTEM_DEF(vote)
 	if(active_vote)
 		win_x = active_vote.win_x
 		win_y = active_vote.win_y
-	show_browser(user, interface(user.client),"window=vote;size=[win_x]x[win_y]")
+	var/datum/browser/popup = new(user, "vote", "Voting Panel", win_x, win_y)
+	popup.set_content(interface(user.client))
+	popup.open(use_onclose = TRUE)
 	onclose(user, "vote", src)
 
 /datum/controller/subsystem/vote/proc/close_panel(mob/user)
```

Comment:
On the line below this, you should delete the browser instead.

Reply:
I don't think it really matters in this case. The browser GCs on its own after the HTML is sent to the client, as far as I am aware.